src/main/java/no/digipost/api/client/representations/archive/ArchiveDocument.java

```diff
@@ -35,10 +35,7 @@
 import java.util.Optional;
 import java.util.UUID;
 
-import static no.digipost.api.client.representations.Relation.ADD_UNIQUE_UUID;
-import static no.digipost.api.client.representations.Relation.GET_ARCHIVE_DOCUMENT_BY_UUID;
-import static no.digipost.api.client.representations.Relation.GET_ARCHIVE_DOCUMENT_CONTENT;
-import static no.digipost.api.client.representations.Relation.GET_ARCHIVE_DOCUMENT_CONTENT_STREAM;
+import static no.digipost.api.client.representations.Relation.*;
```

Comment:
Jeg mener å ha hørt at man prøver å unngå stjerneimporter i digipost-repoer slik at man eksplisitt kan se hvilke biblioteker som er importert. Vet ikke hvor hard man skal være på det, men jeg har i hvert fall konfigurert min IntelliJ til å aldri bruke stjerneimport for java :) Du kan jo vurdere om du vil gjøre det samme.

Reply:
Fikset det, takk!